woocommerce/woocommerce
# [E2E] Remove code related to e2e tests that were previously run using Selenium WebDriver

### All Submissions:


### Changes proposed in this Pull Request:

Remove code related to e2e tests that were previously run using Selenium WebDriver:

- Remove DB (used for the initial setup of the test site on Travis) that is no longer needed;
- Remove the process of building a legacy e2e test site;
- Remove taking screenshots sent to AWS on failed tests (we no longer use the `RUN_E2E` flag + screenshots taking is being replaced by the Slack reporter implemented in https://github.com/woocommerce/woocommerce/pull/25417). 

### How to test the changes in this Pull Request:

1. Run tests locally in headless mode: `npm run test:e2e`;
2. Run tests locally in non-headless mode: `npm run test:e2e-dev`;
3. Make sure tests pass on Travis CI. 

### Other information:




### Changelog entry

Not applicable to e2e tests.
